react/features/base/config/functions.any.js

```diff
@@ -40,6 +40,26 @@ export function createFakeConfig(baseURL: string) {
     };
 }
 
+/**
+ * Selector used to get the meeting region.
+ *
+ * @param {Object} state - The global state.
+ * @returns {string}
+ */
+export function getMeetingRegion(state: Object) {
+    return state['features/base/config'].deploymentInfo.region;
```

Comment:
Can you make sure it will work without deploymentInfo in config.js

Reply:
It's my understanding that we need the region in order to know where to look for the recording.

Is there any other place where we can get the meeting region from? Are there cases where this config info is missing?

Reply:
No, I mean this code will through an error if there is no region in the config. And the default config of people don't have that... 
Make it maybe to return empty string or something and to pass empty string to the request ... To make the code reusable and for other cases, but not totally breaking is what I meant ... Or at least safe guard it, and if there is no deploymentInfo skip everything with some error print ...

Reply:
Ok, got it.